spec/jobs/mailchimp_nonprofit_user_job_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe MailchimpNonprofitUserJob, type: :job do
+  let(:user) {create(:user)}
+  let(:nonprofit) {create(:nonprofit_base)}
+  let(:drip_email_list) {create(:drip_email_list_base)}
+
+  it 'enqueues job when nonprofit user signed up' do 
```

Comment:
We're not testing if the job is enqueued. We're testing that `Mailchimp.signup_nonprofit_user` is called with the arguments when the job is performed